api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTestDesignReport.java

```diff
@@ -1,92 +0,0 @@
-package org.openmrs.module.ugandaemrreports.reports;
```

Comment:
@dbaluku what value does this file serve in the implementation?. if it really does serve  any value then we should give it a different name

Reply:
This file was removed